libsolidity/interface/ReadFile.h

```diff
@@ -37,8 +39,27 @@ class ReadCallback: boost::noncopyable
 		std::string responseOrErrorMessage;
 	};
 
+	enum class Kind
+	{
+		ReadFile,
+		SMTQuery
+	};
+
+	static std::string kindString(Kind _kind)
+	{
+		switch (_kind)
+		{
+		case Kind::ReadFile:
+			return "source";
+		case Kind::SMTQuery:
+			return "smt-query";
```

Comment:
Not fully sure having these two names, but have no other proposal.

Reply:
Do you mean having the names at all or these names specifically?

Reply:
These specific names.